Runner.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace CodeClimate\PHPMD;
+
+use PHPMD\PHPMD;
+use PHPMD\RuleSetFactory;
+use PHPMD\Writer\StreamWriter;
+use PHPMD\Renderer\JSONRenderer;
+
+class Runner
+{
+    private $config;
+    private $server;
+
+    public function setConfig($config)
```

Comment:
Since a `Runner` always needs the `$config` and `$server` members to work properly, thoughts on making them part of the constructor?